modules/twinklewarn.js

```diff
@@ -1202,31 +1203,34 @@ Twinkle.warn.callback.change_category = function twinklewarnCallbackChangeCatego
 	// Trigger custom label/change on main category change
 	Twinkle.warn.callback.change_subcategory(e);
 
-	// Build select menu with jquery.chosen
-	if (!Twinkle.getPref('oldSelect')) {
+	// Use select2 to make the select menu searchable
```

Comment:
Hitting enter when the select menu is focussed does open up the dropdown. This might still be worth doing for maximum accessibility. 

> we can probably be a bit more concise since we know exactly what we're dealing with.

any ideas on how?

Reply:
Anyway, I did some quick playing around and maybe not.  I thought we'd have a good id or other selector to latch on to, but I guess because select2 attaches weirdly it's not super easy (hence the `.closest` I guess)?  It's not too bad I suppose, put it somewhere in Morebits I guess?  enter is scary, and space is non-ideal.

(My initial comment was supposed to say

>It will open automatically when the user hits `<space>` (or `<enter>`!) 

But I left off the ` so they just didn't show!  Now fixed)

Reply:
Maybe setting `dropdownParent` could make it easier?  Played around with it a few days ago with weird results so not sure if it'd be worth it.

Reply:
Dunno about dropdownParent, but do you want the [above hack](https://github.com/azatoth/twinkle/pull/692#discussion_r368061133) added?

Reply:
If you don't mind terribly, I think it'd be worth it 👍